internal/character/danhengimbibitorlunae/ult.go

```diff
@@ -0,0 +1,74 @@
+package danhengimbibitorlunae
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Point                  = "danhengimbibitorlunae-point" // MAvatar_DanHengIL_00_BPCostChange
+	UltPrimary  key.Attack = "danhengimbibitorlunae-ult-primary"
+	UltAdjacent key.Attack = "danhengimbibitorlunae-ult-adjacent"
+	E2          key.Reason = "danhengimbibitorlunae-e2"
+)
+
+var ultHits = []float64{0.3, 0.3, 0.4}
+
+func init() {
+	modifier.Register(Point, modifier.Config{
```

Comment:
Same as the `EnhancheLevel` Modifier. Keeping track of the `Point` via a modifier is not really needed. My suggestion instead would be to just use a variable , give there char some like `Point`variable that you just increment after ult then change it in Attack. Bscly what you are doing with with the `Point` modifier, but change to a variable and not a modifier.